lizard_auth_client/views.py

```diff
@@ -93,12 +99,39 @@ def _sso_post(viewname, payload):
             'key': settings.SSO_KEY,
         }
     )
-    # Check that the request is succesful.
+
+    if return_response:
+        return r
+
+    # Check that the request is succesfull.
+    # catches 4xx/5xx errors
     r.raise_for_status()
+
     # Return the decoded JSON response.
     return r.json()
 
 
+def _new_user_sso_post(viewname, payload):
+    """
+    Wrap __sso_post for new_user specific
```

Comment:
`__` in enkele `_` vervangen.

Docstring: eerste regel moet een enkele regel zijn. En eigenlijk zoiets als "Return dittus en dattus" in plaats van een omschrijving van waarom het gedrag anders is.